lib/ocpp/common/ocpp_logging.cpp

```diff
@@ -69,6 +70,10 @@ MessageLogging::MessageLogging(
                                    "</style>";
             this->html_log_file << "</head><body><table class=\"log\">\n";
         }
+        if (this->log_security) {
+            EVLOG_info << "Logging SecurityEvents to file";
+            this->security_log_file.open(message_log_path + "/" + output_file_name + ".security.log");
```

Comment:
As far as I can see this overwrites the old already existing security log as it does not append?

Reply:
In 1.6 (and 2.0.1) the output_file_name is set to the current datetime: https://github.com/EVerest/libocpp/blob/35195e638790b7a6fc123065b418b10c85c403ab/lib/ocpp/v16/charge_point_impl.cpp#L56

So it's very likely different for each run